rom/laghos.cpp

```diff
@@ -2082,6 +2131,22 @@ int main(int argc, char *argv[])
                              << sqrt(tot_norm) << endl;
                     }
 
+                    if (romOptions.hyperreductionSamplingType == eqp)
```

Comment:
Actually, it will be interesting to be able to see how the energy conservation is like for any kind of hyper-reduction instead of limiting it to EQP. How about we make an option like `calculate_energy`?

Reply:
It turns out this code is not reached, so I will remove it. The energy change is output at the end, around line 2523. This PR added more details about the energy change there.